buildchow/readme.adoc

```diff
@@ -0,0 +1,75 @@
+BuildChow
+=========
+
+BuildChow is the nickname for templates of JBoss Tools and Developer
+Studio jenkins jobs.
+
+By using templates we do not have to rely on jenkins UI to manually update jobs
+or bulk-update in large set of config.xml files.
+
+BuildChow uses http://ci.openstack.org/jenkins-job-builder[Jenkins Job
+Builder](jjb) from Openstack.org.
+
+== Requirements
+
+. Python 2.7+
+. Python PIP
+. Git
+
+On Fedora you get this by runing: `yum install -y python-pip git`
+
+== How to use
+
+First need to install jjb.
```

Comment:
jenkins job builder is what we use, not what we do ;) 

buildchow is just a name for this system - like wonka is name for the build server.